src/main/java/ru/yandex/practicum/filmorate/service/FilmDbService.java

```diff
@@ -47,24 +48,29 @@ public Film put(Film film) {
         return filmStorage.put(film);
     }
 
-    public void putLike(int filmId, int userId) {
+   public void putMark(int filmId, int userId, int mark) {
         if (filmId < 1 || userId < 1) {
             throw new FilmNotFoundException("user or film not found");
         }
-        likeStorage.putLike(filmId, userId);
+        if(mark < 1 || mark > 10){
+            throw new ValidationException("mark not good");
+        }
+       marksStorage.putMark(filmId, userId, mark);
     }
 
-    public void deleteLike(int filmId, int userId) {
+
+    public void deleteMark(int filmId, int userId) {
         if (filmId < 1 || userId < 1) {
```

Comment:
What will happen if userId or filmId > 1 but they are still not valid? No such user exists for instance?